ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -406,10 +458,11 @@ export function PreviewConcierge(props: any) {
               issueDescriptorRequest(path, fetchPrimaryGuestModelComplete);
             }
           } /* else if (type === FETCH_GUEST_MODEL) */ else {
-            if (payload.path?.startsWith('/')) {
-              issueDescriptorRequest(payload.path, fetchGuestModelComplete);
+            const path = payload?.path ? payload.path : guest.path;
```

Comment:
When would there ever not be a path on the payload? This action should always state what model needs loading.